src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -680,7 +680,8 @@ public void testCacheAndCheckWhichAddsNewResourceLocationButKeepsSameCheckerInst
 
         final DynamicalResourceHolderCheck check = new DynamicalResourceHolderCheck();
         final String firstExternalResourceLocation = getPath("checks" + File.separator
-            + "imports" + File.separator + "import-control_one.xml");
+            + "imports" + File.separator + "importcontrol"
+            + File.separator + "InputImportControlOne.xml");
```

Comment:
@romani Right. But should I create it in the scope of this issue or open another PR for creating the folder for Checker? For now I was planning to just move it out to src/test/resources/com/puppycrawl/tools/checkstyle and later make a folder and move it to src/test/resources/com/puppycrawl/tools/checkstyle/checker. What do you suggest?